blocks/gmo-campaign-details/gmo-campaign-details.js

```diff
@@ -453,12 +523,36 @@ export default async function decorate(block) {
                         Adobe Express Mobile App
                     </div>
                 </div>
+                <div class="card scope">
```

Comment:
@mdickson-adbe Can you comment out this section "Feature scope". we are planning to add this next phase 2.0